README.md

```diff
@@ -12,13 +12,17 @@ In order to start the project:
 $ git clone https://github.com/Varenthein/wdp-1811-03.git
 ```
 
-2. Install depedencies
+2. Use npm to run all tasks. In order to run npm install Node.js - you can download it's newest, stable version [from here](https://nodejs.org/en/).
+
+3. Install depedencies
 
 ```sh
 $ cd wdp-1811-03
 $ npm install
 ```
 
+4. Run task `npm run prewatch` in your terminal to get site's production release. It will be located in **dist** directory.
```

Comment:
Jeżeli wspominamy już o prewatch dodałabym także informację o działaniu komendy watch (podgląd "na żywo" + kompilowanie sassów). Niby można wszystko sprawdzić w package.json ale nie jestem pewna czy działanie to jest jakiś ogólnie przyjęty standard.
Gdybym sama trafiła na taki projekt w sieci taka uwaga bardzo ułatwiłaby mi życie. :-)